src/main/java/com/easypost/net/EasyPostResource.java

```diff
@@ -58,6 +59,8 @@ protected enum RequestMethod {
     public static final String EASYPOST_SUPPORT_EMAIL = "support@easypost.com";
     public static final Gson GSON =
             new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)
+                    .serializeNulls()
```

Comment:
Removing. Ultimately unnecessary since the custom HashMap serialization rules override this behavior anyway.

As for users, it won't have any impact. This is for serializing (taking params and turning them into JSON data), not deserializing, so it won't impact the final Java objects/models.